src/app/space/create/deployments/apps/deployment-card.component.html

```diff
@@ -3,8 +3,7 @@
     <div class="card-pf-body">
       <div class="row mini-card-content">
         <div class="col-sm-2">
-          <span class="pficon pficon-ok fa-lg" tooltip="Everything looks okay" placement="top"></span>
-        </div>
+          <deployment-status-icon [cpuDataStream]='cpuStat'></deployment-status-icon>        </div>
```

Comment:
What happened to the indentation of the `</div>` here?

Reply:
my life is in shambles

Reply:
Happened because of an earlier rebase. A fix has been pushed